CMakeLists.txt

```diff
@@ -635,93 +620,112 @@ elseif(${PLATFORM_NAME} STREQUAL Mac OR ${PLATFORM_NAME} STREQUAL Linux)
         COMMENT ${MESSAGE})
 endif()
 
+# Copy Simbody DLLs to build tree.
+# --------------------------------
+# On Windows, we must copy Simbody libraries to the OpenSim build directory
+# so that the tests and examples can run without modifying the PATH
+# (that is, put Simbody's dll's in the same directory as OpenSim's
+# executables and libraries).
+OpenSimCopyDependencyDLLsForWin(Simbody "${Simbody_BIN_DIR}")
+if(WIN32)
+    add_dependencies(Copy_Simbody_DLLs Simbody_CONFIG_check)
+endif()
+
+
 # Directories for Simbody headers and libraries for building.
 # -----------------------------------------------------------
 # TODO no longer necessary; Simbody's library targets provide the 
 # include directories.
```

Comment:
This comment is true, so long as you are using a version of Simbody from within the last month or so. I thought about removing this line, but I did not want to unnecessarily inconvenience the dev team and force everyone to rebuild a newer Simbody.